app/Http/routes.php

```diff
@@ -33,63 +35,61 @@
+        Route::put('/people/{contact}/reminders/{reminder}', 'People\\RemindersController@update')->name('.reminders.update');
+        Route::get('/people/{contact}/reminders/{reminder}/delete', 'People\\RemindersController@destroy')->name('.reminders.delete');
 
         // Tasks
-        Route::get('/people/{people}/tasks/add', ['as' => '.tasks.add', 'uses' => 'PeopleController@addTask']);
-        Route::post('/people/{people}/tasks/store', 'PeopleController@storeTask');
-        Route::get('/people/{people}/tasks/{taskId}/toggle', 'PeopleController@toggleTask');
-        Route::get('/people/{people}/tasks/{taskId}/delete', 'PeopleController@deleteTask');
+        Route::get('/people/{contact}/tasks/add', 'People\\TasksController@create')->name('.tasks.add');
+        Route::post('/people/{contact}/tasks/store', 'People\\TasksController@store')->name('.tasks.store');
+        Route::patch('/people/{contact}/tasks/{task}/toggle', 'People\\TasksController@toggle')->name('.tasks.toggle');
+        Route::get('/people/{contact}/tasks/{task}/delete', 'People\\TasksController@destroy')->name('.tasks.delete');
 
         // Gifts
-        Route::get('/people/{people}/gifts/add', ['as' => '.gifts.add', 'uses' => 'PeopleController@addGift']);
-        Route::post('/people/{people}/gifts/store', 'PeopleController@storeGift');
-        Route::get('/people/{people}/gifts/{giftId}/delete', 'PeopleController@deleteGift');
+        Route::get('/people/{contact}/gifts/add', 'People\\GiftsController@create')->name('.gifts.add');
+        Route::post('/people/{contact}/gifts/store', 'People\\GiftsController@store')->name('.gifts.store');
+        Route::get('/people/{contact}/gifts/{gift}/delete', 'People\\GiftsController@destroy')->name('.gifts.delete');
 
         // Debt
-        Route::get('/people/{people}/debt/add', ['as' => '.debt.add', 'uses' => 'PeopleController@addDebt']);
-        Route::get('/people/{people}/debt/{debtId}/edit', ['as' => '.debt.edit', 'uses' => 'PeopleController@editDebt']);
-        Route::post('/people/{people}/debt/{debtId}/update', ['as' => '.debt.update', 'uses' => 'PeopleController@updateDebt']);
-        Route::post('/people/{people}/debt/store', 'PeopleController@storeDebt');
-        Route::get('/people/{people}/debt/{debtId}/delete', 'PeopleController@deleteDebt');
+        Route::get('/people/{contact}/debt/add', 'People\\DebtController@create')->name('.debt.add');
+        Route::post('/people/{contact}/debt/store', 'People\\DebtController@store')->name('.debt.store');
+        Route::get('/people/{contact}/debt/{debt}/edit', 'People\\DebtController@edit')->name('.debt.edit');
+        Route::put('/people/{contact}/debt/{debt}', 'People\\DebtController@update')->name('.debt.update');
+        Route::get('/people/{contact}/debt/{debt}/delete', 'People\\DebtController@destroy')->name('.debt.delete');
```

Comment:
Done.

This was oversight on my part, sorry. Adding App\Debt to RouteServiceProvider fixes it.